### ISSUE TITLE
BUG:Logo Missing and Alt Text Displayed in Top Left Corner

### ISSUE THREAD

**Issue by Logesh-waran2003** [dakshsinghrathore/ISRO-web#91]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Issue Description:
On the [privacy page], there is a problem with the logo in the top left corner. Instead of displaying the logo, it's showing the alt text associated with the image. This issue impacts the visual aesthetics and usability of the page.

Steps to Reproduce:

Navigate to the [https://dsr-isrowebapp.netlify.app/privacy] at the bottom of the home page.
Please look over the top left corner of the page where the logo is supposed to be displayed.

Expected Behavior:
The logo should be displayed as intended, providing users with a visually appealing and informative element that redirects to the home page.

Actual Behavior:
The logo is missing, and the alt text is shown, and it redirects to the same page, which is not the desired behavior.

Additional Information:

The browser and version used (Brave and Google chrome).

Screenshots:

![Screenshot from 2023-10-15 15-57-22](https://github.com/dakshsinghrathore/ISRO-web/assets/81814503/73bffecb-4644-42d9-882f-3e0f8f528d4f)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a Hacktoberfest'23 contributor
- [X] I want to work on this issue

**Comment by dakshsinghrathore** [comment]
This issue has been resolved, kindly work on some other issues, if you don't find one, wait will update the issues and you can work on them !
